app/apis/v1/record_sets_api.rb

```diff
@@ -0,0 +1,76 @@
+module V1
```

Comment:
In the domain, the cases are totally different from each other.
So, I recommend you use a different class to describe it.